lib/turn.rb

```diff
@@ -0,0 +1,30 @@
+class Turn
+  attr_reader :guess, :card
+  def initialize(guess, card)
+    @guess = guess
+    @card = card
+
+  end
+
+
+
+  def correct?
+    if @guess == card.answer
+      true
+    elsif @guess != card.answer
+      false
```

Comment:
While this code works, and could be used without issue I do want to point out that there is a bit of redundancy here.

All three of the following methods do the exact same thing with less and less code for each one.

```ruby
def correct?
  if @guess == card.answer
    true
  elsif @guess !- card.answer
    false
  end
end
```

```ruby
def correct?
  if @guess == card.answer
    return true
  end

  false
end
```

```ruby
def correct?
  @guess == card.answer #<=== conditional statements return booleans by default, no need to say "if something return true, else return false"
end
```

Again, the method works just fine as written, but just wanted to point it out.